usersDb.js

```diff
@@ -0,0 +1,69 @@
+const pgp = require('pg-promise')()
+const connectionString = process.env.DATABASE_URL || 'postgres://localhost:5432/contacts'
+const db = pgp(connectionString)
+
+const users = {}
+
+users.addNewUser = (email, password) => {
+  return db.query(`
+    INSERT INTO 
+      users (email, password)
+    VALUES
+      ($1, $2)
+  `,
+  [
+    email,
+    password
+  ])
+  .catch(error => error)
+}
+
+users.findUser = (email) => {
+  return db.query(`
+    SELECT
+      *
+    FROM
+      users
+    WHERE
+      email = $1
+  `,
+  [
+    email
+  ])
+  .then(data => data)
```

Comment:
instead of calling it `data`, you rather call name the variable `users` as it indicates what the variable contains. 
Also since the function is called `findUser`, you should be returning the first element of the results returned.